stable/instana-agent/values.yaml

```diff
@@ -68,6 +68,10 @@ agent:
   # agent.redactKubernetesSecrets sets the INSTANA_KUBERNETES_REDACT_SECRETS environment variable.
   # redactKubernetesSecrets: null
 
+  # agent.host.repository sets a host path to be mounted as the agent maven repository (for debugging or development purposes)
+  host:
+    repsitory: null
```

Comment:
This should be `repository` rather than `rep_sitory`

Reply:
Hah - I guess that means when I "tested" this I didn't actually test it hahah